dds/idl/gen_util.h

```diff
@@ -0,0 +1,33 @@
+
+#ifndef OPENDDS_IDL_GEN_UTIL_H
+#define OPENDDS_IDL_GEN_UTIL_H
+
+#include "be_extern.h"
+
+namespace {
```

Comment:
Remove the `namespace` here.  This is for two reasons:
- in general, anonymous namespaces shouldn't be in headers
- since `opendds_idl` is not a library, there is no chance of name collision with end-uesr code